packages/toolkits/pro/template/server/nestJs/src/user/user.controller.ts

```diff
@@ -44,4 +46,16 @@ export class UserController {
   ) {
     return this.userService.getAllUser(paginationQuery);
   }
+
+  @Patch('/admin/updatePwd')
+  @Permission('user::update')
```

Comment:
```suggestion
  @Permission('user::password::force-update')
```
需要改成这样，否则有[user::update]权限的用户也会被允许访问该接口